server/src/__tests__/completions.test.ts

```diff
@@ -197,6 +197,52 @@ describe('On Completion', () => {
     expect(result).toEqual([])
   })
 
+  it('provides recipe (.bb) suggestions in the string conent of certain variable assignments', async () => {
```

Comment:
More accurately, we are providing completion for packages here, not recipes. (one recipe can provide multiple packages).
 1. You could remove the .bb from the completion tooltip
 2. I think we have a more accurate packages list instead of recipes list? If we do you should use that as completions

Reply:
1. When the completion is shown, .bb should not be printed in the tooltip illustrated here. once we hit enter, it is already properly removed so it shouldn't be showed
![image](https://github.com/yoctoproject/vscode-bitbake/assets/19970889/d14bd130-ac28-46fe-bbbe-740f301dd764)
2. We don't, I think we properly handle packages through the old deep scan which we have refactored now. Nothing more to do on that topic.

Reply:
Done.